tests/Providers/AbstractProviderTest.php

```
@@ -0,0 +1,24 @@
+
+namespace DrevOps\EnvironmentDetector\Tests\Providers;
+
+use DrevOps\EnvironmentDetector\Providers\AbstractProvider;
+use DrevOps\EnvironmentDetector\Tests\TestBase;
+use PHPUnit\Framework\Attributes\CoversClass;
+
+#[CoversClass(AbstractProvider::class)]
+class AbstractProviderTest extends TestBase {
+
+  public function testDataWithEmptyPrefixes(): void {
+    $empty_prefixes_provider = $this->mockProvider(
```

PHPMD (2.15.0)

The variable $empty_prefixes_provider is not named in camelCase. (undefined)

(CamelCaseVariableName)